internal/templates/internal/home.html

```diff
@@ -1,5 +1,16 @@
 {% extends "internal/base.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block head %}
+    <link rel="stylesheet" href="{% static "internal/css/home.css" %}">
+{% endblock %}
 
 {% block body %}
+   <div class="main">
+
+    <h1>{% trans "MAKE NTNU Internal" %}</h1>
+
+    </div>
 
 {% endblock %}
```

Comment:
It would be great if you could clean up this formatting 🙂 I can recommend regularly using a tool - like PyCharm's "Reformat Code" feature (Ctrl+Alt+L by default) - to keep a consistent code format.

```suggestion
    <div class="main">

        <h1>{% trans "MAKE NTNU Internal" %}</h1>

    </div>
{% endblock %}
```